### PR TITLE
LPD-31028 Technical Task | Fix test failures in JournalArticleModelDocumentContributorTest

### PR DESCRIPTION
**Pull request by Mikellorza**
# What is this trying to solve?

I run the tests many times and always the run has been successful. This is not easy to determine why the tests are failing  once in a while. I reviewed it but I was not able to conclude nothing because I am not seeing any pattern. I suppose that for any reason when we are calling to journalArticle.getDDMFormValues(); in [this line](https://github.com/liferay/liferay-portal/blob/796cae0a25770283167b1c1f11bc73d5390f9f09/modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/search/spi/model/index/contributor/JournalArticleModelDocumentContributor.java#L76) is returning a null object and for that some tests are failing. I think that the problem is in[ this line  ](https://github.com/liferay/liferay-portal/blob/518efbd11c23baab9a10494ef521f28b08dc5f8e/modules/apps/journal/journal-service/src/main/java/com/liferay/journal/model/impl/JournalArticleImpl.java#L275) and for that reason we are getting a null object but I am not sure 100%, I am assuming things and for that reason I am adding some improves to determine why is failing  in future following executions. 

https://liferay.atlassian.net/browse/LPD-29856

# How am I fixing it?

Add some improves to determine in the future why the tests are failing.

# How can I test it?

Execute the test.

### CODE REVIEW

File: modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/internal/search/spi/model/index/contributor/test/JournalArticleModelDocumentContributorTest.java [hunk 1]
```diff
@@ -129,6 +129,9 @@ private String _getDDMIndexerContent(String languageId) throws Exception {
 	private Document _getDocument() {
 		DocumentImpl documentImpl = new DocumentImpl();
 
+		Assert.assertNotNull(_journalArticle.getDDMFormValues());
+		Assert.assertNotNull(_journalArticle.getDDMFormValues());
```
Comment (adolfopa) [inline modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/internal/search/spi/model/index/contributor/test/JournalArticleModelDocumentContributorTest.java:133]: Why checking it twice? Is this on purpose?

**Review by adolfopa — CHANGES_REQUESTED** [review]

File: modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/internal/search/spi/model/index/contributor/test/JournalArticleModelDocumentContributorTest.java (hunk 1 above)
Comment (Mikellorza) [reply modules/apps/journal/journal-test/src/testIntegration/java/com/liferay/journal/internal/search/spi/model/index/contributor/test/JournalArticleModelDocumentContributorTest.java]: My mistake. I've just sent again fixing it. 

thanks @adolfopa

**Comment by Mikellorza** [comment]
ci:test:sf

**Comment by Mikellorza** [comment]
@adolfopa  if you see something to fix it I appreciate.

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**